docs/schemaValidatedComponent.md

```diff
@@ -0,0 +1,159 @@
+# `gpii.schema.component`
+
+A "schema validated component" is one whose entire structure (including all options) is validated as part of its
```

Comment:
Hard to say that it is the "entire" structure - since the only bits of it that will be validated are the bits a schema has been written for!

Reply:
I ended up with better wording in the summary in the gss.md file and reused key bits to improve this.